### ISSUE TITLE
Implement pagination for war report query

### ISSUE THREAD

**Issue by krishna-koushik** [krishna-koushik/azoth-empire#10]
Limit 25 wars per page
Provide the following sort
** Ascending and Descending order by attacker
** Ascending and Descending order by defender
** Ascending and Descending order by warTime (default latest war first)
** Ascending and Descending order by warType
Provide information about current page numbers.

**Comment by Airaken** [comment]
I found certain things:
- in the structure for Attacker the `guild` field can be a type Guild we could keep another `guildName` field and leave `guild` to access more data in the future.
- to sort by defender there is no field in the database with this data (or at least I did not see it). 

on the other hand when making the War pagination I noticed that it uses practically the same structure as for Player, basically it is repeated code I think we should take a look at `application-model` and leave it a little more general so that the pagination can be applied regardless of the type.
